nibabel/volumeutils.py

```diff
@@ -35,8 +35,8 @@
     DT = ty.TypeVar('DT', bound=np.generic)
 
 sys_is_le = sys.byteorder == 'little'
-native_code = sys_is_le and '<' or '>'
-swapped_code = sys_is_le and '>' or '<'
+native_code = (sys_is_le and '<') or '>'
+swapped_code = (sys_is_le and '>') or '<'
```

Comment:
Ternary expressions feel cleaner to me. Also seems easier to spot an error if we do this on one line:

```suggestion
native_code, swapped_code = ('<', '>') if sys_is_le else ('>', '<')
```